### PR TITLE
nerfs parrots

### PR DESCRIPTION
**Pull request by Iamgoofball**
i feel that "grab the object from a person's hands and put it into the contents of the parrot no cooldown no fail chance" is a little overpowered

### CODE REVIEW

File: code/modules/mob/living/simple_animal/parrot.dm [hunk 1]
```diff
@@ -701,11 +701,15 @@
...
-			visible_message("[src] grabs [held_item] out of [C]'s hand!", "<span class='notice'>You snag [held_item] out of [C]'s hand!</span>", "You hear the sounds of wings flapping furiously.")
-			return held_item
+			if(prob(50))
+				C.unEquip(stolen_item)
+				held_item = stolen_item
+				stolen_item.loc = src
+				visible_message("[src] grabs [held_item] out of [C]'s hand!", "<span class='notice'>You snag [held_item] out of [C]'s hand!</span>", "You hear the sounds of wings flapping furiously.")
+				return held_item
+			else
+				visible_message("[src] fails to grab [held_item] out of [C]'s hand!", "<span class='notice'>You fail to snag [held_item] out of [C]'s hand!</span>", "You hear the sounds of wings flapping furiously.")
```
Comment (Thunder12345) [inline code/modules/mob/living/simple_animal/parrot.dm]: Should this be [stolen_item]? Don't see held_item being set anywhere there.

Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/parrot.dm]: it should.

File: code/modules/mob/living/simple_animal/parrot.dm [hunk 2]
```diff
@@ -701,11 +701,15 @@
 			stolen_item = C.r_hand
 
 		if(stolen_item)
-			C.unEquip(stolen_item)
-			held_item = stolen_item
-			stolen_item.loc = src
-			visible_message("[src] grabs [held_item] out of [C]'s hand!", "<span class='notice'>You snag [held_item] out of [C]'s hand!</span>", "You hear the sounds of wings flapping furiously.")
-			return held_item
+			if(prob(50))
```
Comment (Stickymayhem) [inline code/modules/mob/living/simple_animal/parrot.dm]: This shouldn't be here. Clearly an error from a robusted nerd.

**Comment by Iamgoofball** [comment]
eh fuck it this whole proc needs reworked

**Pull request closed without merging**